src/app/qgsclipboard.cpp

```diff
@@ -309,6 +309,11 @@ QgsFeatureList QgsClipboard::copyOf( const QgsFields &fields ) const
   QString text = cb->text( QClipboard::Clipboard );
 #endif
 
+  if (text.endsWith('\n'))
```

Comment:
For windows users we better check for and remove carriage returns too

Reply:
I was actually "watching" the Windows clipboard as I was testing (with FreeClipViewer): "\n" alone did fix the problem when the last two characters were `CRLF` in the clipboard. I assumed Qt5 was normalizing newlines to `LF` as it read from the clipboard. I will double check and modify as necessary.